openedx_external_enrollments/external_enrollments/mit_hz_external_enrollment.py

```diff
@@ -61,7 +61,7 @@ def _get_bearer_token(self):
             )
         else:
             if not response.ok:
-                log_details['response'] = response.text()
+                log_details['response'] = response.text
```

Comment:
Is there any reason why we are not using response.json() @luismorenolopera ?

Reply:
Yes.  the MIT Horizon API when it fails does not return a JSON, it returns an HTML file